all-in-one/jaeger-all-in-one-template.yml

```diff
@@ -12,11 +12,6 @@
 # the License.
 #
 
-# This template uses Jaeger with in-memory storage with limited functionality
```

Comment:
We have a readme with this info. I would like to keep templates as clean as possible. If you want I can bring it back.